src/Microsoft.AspNet.Identity/UserOptions.cs

```diff
@@ -11,37 +11,12 @@ namespace Microsoft.AspNet.Identity
     public class UserOptions
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="UserOptions"/> class.
-        /// </summary>
-        public UserOptions()
-        {
-            //User.RequireUniqueEmail = true; // TODO: app decision?
-        }
-
-        /// <summary>
-        /// Gets or sets the regular expression used to validate user names.
-        /// </summary>
-        /// <value>
-        /// The regular expression used to validate user names.
-        /// </value>
-        /// <remarks>
-        /// As regular expressions can be subject to Denial of Service attacks, depending on their complexity and user input,
-        /// validation via regular expressions will timeout and fail after the value set in the <see cref="UserNameValidationRegexTimeout"/>
-        /// property.
-        /// </remarks>
-        public string UserNameValidationRegex { get; set; } = "^[a-zA-Z0-9@_\\.]+$";
-
-        /// <summary>
-        /// Gets or sets the timeout value used after which user name validation via the <see cref="UserNameValidationRegex"/> will fail if it has
-        /// not completed.
+        /// Gets or sets the list of allowed characters in the username used to validate user names.
         /// </summary>
         /// <value>
-        /// The timeout value used after which user name validation via the <see cref="UserNameValidationRegex"/> will fail if it has not completed.
+        /// The list of allowed characters in the username used to validate user names.
         /// </value>
-        /// <remarks>
-        /// The default value is 20 milliseconds.
-        /// </remarks>
-        public TimeSpan UserNameValidationRegexTimeout { get; set; } = new TimeSpan(0,0,0,0,20);
+        public string AllowedUserNameCharacters { get; set; } = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-._@";
```

Comment:
I don't see any special characters in your username :). Or did it get englisized?